q2cli/tools.py

```diff
@@ -55,6 +70,9 @@ def export_data(path, output_dir):
               help='The format of the data to be imported. If not provided, '
                    'data must be in the format expected by the semantic type '
                    'provided via --type.')
+@click.option('--show-importable-types', is_flag=True, is_eager=True,
```

Comment:
For the `help` string, can you make it a little more descriptive, something like "Show the semantic types that can be supplied to --type to import data into an artifact." ?